# Задание

В файле main.py содержаться задания. Например, Три первых задания выглядят таким образом:

```python
class Lesson7():
    """ 
      Задание-01

      Создайте переменную с именем `my_var` и присвойте ей значение любой строки
    """
    """ После этой строчки вы должны написать код Задания-01 """
    my_var = "Zf"

    """ Конец Задания-01. Код к данному заданию должет быть только выше этой строчки """

    ####################################################################################

    """ 
      Задание-02
      Создайте константу с именем 'MY_CONST` и присвойте ей значение любого целого числа
    """
    """ После этой строчки вы должны написать код Задания-02 """

    """ Конец Задания-02. Код к данному заданию должет быть только выше этой строчки """

    ####################################################################################

    """ 
      Задание-03
      Напечатайте в консоль "Привет, мир, я Зомби!"
    """

    def test_03(self):
        """ После этой строчки вы должны написать код Задания-03 """

        """ Конец Задания-03. Код к данному заданию должет быть только выше этой строчки """

    ########################################################################################
```

Т.е. Между строчками комментария вы для каждого задания должны написать свой код.
Писать свой код нужно только между указанными для каждого задания строчками.
Везде будут отступы! Учитывайте это! В первых двух заданиях 2 отступа. После - 4 отступа!
Не модифицируйте комментарии!

Заданий будет несколько. Выполнять их надо последовательно.

Чтобы протестировать правильно ли вы выполнили задание вы должны в КОНСОЛЕ выполнить команду `python test.py` (Без кавычек естественно).

Если задание выполнено правильно то вы увидите что-то типа:

```s
PS D:\work\serious-gregor> python test.py
..
----------------------------------------------------------------------
Ran 2 tests in 0.026s

OK
```

Если же вы что-то сделали не так, то вы увидите ошибку. Например такую:

```s
PS D:\work\serious-gregor> python test.py
.F
======================================================================
FAIL: test_02 (__main__.Test_Lesson7)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test.py", line 16, in test_02
    5), "Тип вашей константы не целое число!")
AssertionError: <class 'str'> != <class 'int'> : Тип вашей константы не целое число!

----------------------------------------------------------------------
Ran 2 tests in 0.001s

FAILED (failures=1)
```

Вы можете в середине увидеть сообщение что у вас неверно.

